packages/calypso-e2e/src/lib/pages/plans-page.ts

```diff
@@ -17,31 +22,41 @@ const selectors = {
 	activeNavigationTab: ( tabName: PlansPageTab ) =>
 		`.is-selected.section-nav-tab:has-text("${ tabName }")`,
 
-	actionButton: ( { plan, buttonText }: { plan: Plan; buttonText: PlanActionButton } ) => {
+	actionButton: ( { plan, buttonText }: { plan: LegacyPlans; buttonText: PlanActionButton } ) => {
 		const viewportSuffix = envVariables.VIEWPORT_NAME === 'mobile' ? 'mobile' : 'table';
 		return `.plan-features__${ viewportSuffix } >> .plan-features__actions-button.is-${ plan.toLowerCase() }-plan:has-text("${ buttonText }")`;
 	},
 
-	// Plans view
-	plansGrid: '.plans-features-main',
+	// Legacy plans view
+	legacyPlansGrid: '.plans-features-main',
+
+	// Overhauled plans view
+	overhauledPlansGrid: 'div.plans > table',
```

Comment:
Did you need this selector? Or is it for future use? I didn't see it used anywhere, so wanted to double check